maintainers/working_groups/learning_journey/README.md

```diff
@@ -28,42 +28,9 @@ Membership is open to anyone that wants to contribute.
 Meetings will be held every week to ensure that progress can be made between meetings.
 
 ## Immediate projects
-### Decide on documentation structure
-This project will decide on a high level structure for the documentation so that contributors know the audience they are speaking to and where their contributions fit into the Nix documentation landscape.
 
-The documentation will follow the [Diataxis](https://diataxis.fr) framework to the degree that it makes sense. This project will produce a proposal to be submitted to the Documentation Team.
```

Comment:
Why remove the reference to Diataxis?

Reply:
Just an oversight. I removed the whole section to remove the outline and then wrote a little bit about the project. I'll add it back in.